job/validators.py

```diff
@@ -390,13 +397,13 @@ def cross_clean_pv(pvmodel):
 
             if self.models["Generator"].__getattribute__("replacement_year") == None:
                 if self.models["Settings"].off_grid_flag==False:
-                    self.models["Generator"].replacement_year = 25
+                    self.models["Generator"].replacement_year = 25 # TODO: set to analysis_years
                 else:
                     self.models["Generator"].replacement_year = 10
 
             if self.models["Generator"].__getattribute__("replace_cost_per_kw") == None:
                 if self.models["Settings"].off_grid_flag==False:
-                    self.models["Generator"].replace_cost_per_kw = 410.0
+                    self.models["Generator"].replace_cost_per_kw = 410.0 # TODO: set to 0
```

Comment:
@rathod-b this should be 0 to align with Julia Pkg default (replacement costs not considered by default for on-grid scenarios)